articles/article-125-euler-path.md

```diff
@@ -0,0 +1,18 @@
+<!-- BEGIN TITLE -->
+# How to navigate in Königsberg
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+The city of Königsberg is split by a river into 4 land masses. Those are connected by 7 lovely bridges that invite to a walk. The best route is of course the one which crosses each bridge exactly once.
```

Comment:
when using numbers it is usually best practice to spell then out when they are greater then 100 (I have seen exceptions to this rule if it is part of a name like a street)

Reply:
The city of Königsberg is split by a river into four separate land masses. They are connected by seven lovely bridges, which are pedestrian friendly.

Reply:
The best route would be to cross each bridge exactly once. (I am correctly for something more succinct)